plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/config.py

```diff
@@ -50,6 +50,7 @@ class AxConfig:
     experiment: ExperimentConfig = ExperimentConfig()
     client: ClientConfig = ClientConfig()
     params: Dict[str, Any] = field(default_factory=dict)
+    is_noisy: bool = True
```

Comment:
Can you add a comment here?
maybe with a link to the Ax docs if there is anything good to link to?